### ISSUE TITLE
Who are you? Link back to you!

### ISSUE THREAD

**Issue by altsheets** [lepht/licecap#17]
I suggest (a) new button(s):  

* **Info**, or **About**, or **Homepage**, and/or   
* **Sourcecode**

which  
* tell me a bit about the program, and  
* bring me to   
** http://www.cockos.com/licecap/, and perhaps even to  
** http://github.com/lepht/licecap  

Why? At first, I was a bit lost when I tried to find your address to leave my enthusiastic [testimonial](https://github.com/lepht/licecap/wiki/User-testimonials). In your *whatsnew.txt* you even link to another github - but not to yourself! And the *license.txt* is pure GPL, also no link back to you. And the rightclick-on- *licecap.exe* ...Properties...Details... just says "Application" and a date - but nothing else.

TL;DR: Make it easier for your users ... to celebrate you!
